### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by yiwenxue**
Update ReadMe
 * how to build the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,39 @@
...
 │   └── native/external (<a href="https://github.com/cocos/cocos-engine-external/tree/v3.5">3.5</a>)
 └── test-gfx (<a href="https://github.com/cocos-creator/test-gfx/tree/master">master</a>)
 </pre>
+
+### web
+
+Enter ./engine folder and run the following commands
+```bash
+    $ ./util.sh build # build the ts engine.
+    $ ./util.sh update # build the ts engine.
```
Comment (minggo) [inline README.md]: two commands to build ts engine?

**Comment by minggo** [comment]
How to build on macOS?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,39 @@
...
 ├── cocos-engine (<a href="https://github.com/cocos/cocos-engine/tree/v3.5.0">3.5.0</a>)
 │   └── native/external (<a href="https://github.com/cocos/cocos-engine-external/tree/v3.5">3.5</a>)
 └── test-gfx (<a href="https://github.com/cocos-creator/test-gfx/tree/master">master</a>)
 </pre>
+
+### web
+
+Enter ./engine folder and run the following commands
+```bash
+    $ ./util.sh update       # build the project.
```
Comment (minggo) [inline README.md:19]: command is update, the operation is building?

Comment (yiwenxue) [reply README.md:19]: Yes, update will build the project and then copy the result to `engine/web-desktop`.

**Comment by yiwenxue** [comment]
> How to build on macOS?

I am not sure about this as I don't have IOS/Mac device. @hana-alice should be familiar with this issue.

**Pull request merged**